src/og/renderer/RendererEvents.js

```diff
@@ -566,20 +566,20 @@ class RendererEvents extends Events {
     onTouchMove(event) {
         var ts = this.touchState;
         ts.clientX = event.touches.item(0).clientX - event.offsetLeft;
-        ts.clientY = event.touches.item(0).clientY - event.offsetTop;
+        ts.clientY = event.touches.item(0).clientY - event.offsetTop;
 
-        var h = this.renderer.handler;
-
-        ts.x = ts.clientX * h.pixelRatio;
-        ts.y = ts.clientY * h.pixelRatio;
+        var h = this.renderer.handler;
+
+        ts.x = ts.clientX * h.pixelRatio;
+        ts.y = ts.clientY * h.pixelRatio;
 
         ts.nx = ts.x / h.canvas.width;
-        ts.ny = ts.y / h.canvas.height;
+        ts.ny = ts.y / h.canvas.height;
 
         ts.sys = event;
         ts.moving = true;
-        this._dblTchBegins = 0;
-        this._oneTouchStart = false;
+        // this._dblTchBegins = 0;

```

Comment:
Does double touch work fine with it?

Reply:
It works, but not fine.
If the user wants to quickly swipe the globe with one finger multiple times(in doubleTouchDelay: 550ms), it will become a double touch zoomIn.
It is best to add the judgment of the finger movement distance, and add the judgment of the shorter movement distance before double touch,  to distinguish between single-touch sliding and double-touch zoomIn.
Of course, we can also merge this code first, and then optimize it later.

Reply:
Let me submit another optimized version in the next few days.
This PR can be discarded first.
The next version will also add two-finger zoomin and zoom out, I'm working on the code.